core/build.sbt

```diff
@@ -14,9 +14,12 @@ libraryDependencies ++= Seq(
   "com.twitter"       %% "algebird-core"        % "0.8.0",
   "org.fusesource"     % "sigar"                % "1.6.4" classifier("native") classifier("") exclude("log4j", "log4j"),
   "org.slf4j"          % "log4j-over-slf4j"     % "1.7.+", // SIGAR requires the log4j legacy API
-  "oncue.journal"     %% "core"                 % "2.2.1"
+  "oncue.journal"     %% "core"                 % "2.2.1",
+  "org.spire-math"    %% "spire"                % "0.11.0"
```

Comment:
Specifically for the `Natural` type. It's a crime that this type isn't in the standard library, but here we are. This is in an effort to make the `Counter` API more safe.